packages/mobile/src/send/SendAmount.tsx

```diff
@@ -202,9 +209,26 @@ function SendAmount(props: Props) {
     }
   }, [props.route, localCurrencyCode, localCurrencyExchangeRate, dollarAmount])
 
+  const onReviewButtonPressed = async () => {
+    setReviewButtonToPressed(true)
+    // Wait until verification status is known to proceed
+    while (recipientVerificationStatus === RecipientVerificationStatus.UNKNOWN) {
+      await wait(250)
```

Comment:
We use `sleep` from `sleep-promise` elsewhere - any particular reason to use wait here instead?

Reply:
Gotcha - will use that instead